ingenialink/ethercat/network.py

```diff
@@ -153,11 +184,76 @@ def disconnect_from_slave(self, servo: EthercatServo) -> None:  # type: ignore [
 
         """
         servo.stop_status_listener()
+        if not self._change_nodes_state(servo, pysoem.INIT_STATE):
+            logger.warning("Drive can not reach Init state")
```

Comment:
there are still references to drive instead of slave or node

Reply:
I think we should stick to "slave"